lib/pathway/rspec/matchers/form_schema_helpers.rb

```diff
@@ -21,6 +21,14 @@ def not_defined_list
         "#{as_list(not_defined)} #{were_was(not_defined)} not defined" if not_defined.any?
       end
 
+      def accepting_null_list
+        "#{as_list(null_value_allowed)} #{were_was(null_value_allowed)} accepting null values" if null_value_allowed.any?
```

Comment:
Here it should  be `if @not_allowing_null_values && null_value_allowed.any?`, because if the user if not testing for it we shoudn't report anything about it.